RMStore/RMStore.m

```diff
@@ -133,16 +133,16 @@ @implementation RMStore {
     NSMutableDictionary *_addPaymentParameters; // HACK: We use a dictionary of product identifiers because the returned SKPayment is different from the one we add to the queue. Bad Apple.
     NSMutableDictionary *_products;
     NSMutableSet *_productsRequestDelegates;
-    
+
```

Comment:
Please refrain from making style changes. It makes the PR harder to review, and -unless done globally- makes the code style heterogenous.